pgmuvi/lightcurve.py

```diff
@@ -330,7 +330,29 @@ def fit(self, model = None, likelihood = None, num_mixtures = 4,
 
         #Now we're going
 
-        
+    def plot_psd(self, means, freq, scales, weights):
```

Comment:
Would it be possible to add a keyword to control whether to display the plots or return them (so a user could customise them if they wanted)? Maybe something like

``` suggestion
    def plot_psd(self, means, freq, scales, weights, show=True):
```

and then line 342 would also be modified (see other suggestion below)